pkg/ccl/changefeedccl/testfeed_test.go

```diff
@@ -127,16 +129,26 @@ type sinklessFeed struct {
 
 	conn           *pgx.Conn
 	rows           pgx.Rows
+	cancel         context.CancelFunc
 	latestResolved hlc.Timestamp
 }
 
 var _ cdctest.TestFeed = (*sinklessFeed)(nil)
 
+func deadline() time.Duration {
+	if util.RaceEnabled {
+		return time.Minute
```

Comment:
Race can be _really_ slow in CI sometimes. I might even consider something like 5 minutes.

Reply:
Ack.